OpenQA/Isotovideo/Utils.pm

```diff
@@ -64,4 +65,27 @@ sub checkout_git_repo_and_branch {
     $bmwqemu::vars{$dir} = File::Spec->rel2abs($local_path);
 }
 
+=head2
+
+Helper function to create backward compatible function arguments when moving
+from positional arguments to named one.
+
+    compat_args( $hash_ref_defaults, $arrayref_old_fixed, [ $arg1, $arg2, ...])
+
+A typical call would look like:
+
+    my %args = compat_args({timeout => 60, .. }, ['timeout'], @_);
+=cut
+sub compat_args {
```

Comment:
I like that you created a separate function for this recurring and not that simple code. Maybe it is worth creating a dedicated unit test for it.

Reply:
I'm on it now.

Reply:
done for compat_args(). Need to do it for *_run() calls